domains/clicpow.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ClicpoW",
+        "email": "clicpowytb@gmail.com"
+    },
+    "record": {
+        "CNAME": ["https://mlkfkqd.pages.dev/"],
```

Comment:
Invalid CNAME record (must be a string, not an array)

Reply:
also CNAME can't have `https://` or `/`

Reply:
it works

Reply:
thats because maintainer updates the file for you

Reply:
yea i saw thanks